components/director/internal/domain/formationtemplate/resolver_test.go

```diff
@@ -180,7 +180,7 @@ func TestResolver_FormationTemplates(t *testing.T) {
 			First: &first,
 			FormationTemplateService: func() *automock.FormationTemplateService {
 				svc := &automock.FormationTemplateService{}
-				svc.On("List", txtest.CtxWithDBMatcher(), first, after).Return(&formationTemplateModelPage, nil)
```

Comment:
Change the `mock.Anything` with the actual value(I think it should be `nil`). This applies to all of the resolver and service tests.

Reply:
With `nil` the signature does not match and the tests fail with: 
`1: FAIL:  (*string=<nil>) != (<nil>=<nil>)`

One way to remove `mock.Anything` would be:
`var nilString *string`
then replace `mock.Anything` with `nilString`. Should I change them to that?

Reply:
I think it would be better wit nilStr.